decidim/decidim
# Refactor Conference diploma to use hexaPDF

#### :tophat: What? Why?
In this PR we remove the wicked_pdf usage from the last known place, in the Conference diploma. 

#### Testing
1. Login to admin & register to a conference 
2. Visit that conference admin's panel and configure the diploma parameters 
3. make sure to confirm the presence of the user 
4. when send attendence conference diploma buton is available send it 
5. You should receive a diploma on email. 

Note to review-er, because we want to have the diploma translatable, and the way HexaPDF handles the strings, i had to remove some bolded text from the line "This is to certify ... " 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
